errors/custom_error.js

```diff
@@ -0,0 +1,13 @@
+
+class CustomAPIError extends Error{
```

Comment:
We don't need empty line at the top
```suggestion
class CustomAPIError extends Error{
```